msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase99656.java

```diff
@@ -103,6 +106,8 @@ public void handleUserInteraction() {
         silentLatch.await(TokenRequestTimeout.SILENT);
 
         // second interactive request
+        // wait about a minute here to throttle usage of AUTO MFA account
+        Thread.sleep(TimeUnit.MINUTES.toMillis(1));
```

Comment:
ThreadUtils.sleepSafely()?

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1251/commits/fd1a43460545e3ca9127b24b0f85b96cfc8c9022